app/controllers/wpcc/your_details_controller.rb

```diff
@@ -8,11 +10,13 @@ def new
 
     def create
       @your_details_form = Wpcc::YourDetailsForm.new(your_details_form_params)
+      @your_details_form = present(@your_details_form)
```

Comment:
The reassignment of @your_details_form looks a little odd, wdyt 

`@your_details_form = present(Wpcc::YourDetailsForm.new(your_details_form_params))`